doc/README.md

```diff
@@ -0,0 +1,11 @@
+Documentation
+=============
+
+This directory contains documentation about various components in OpenJ9
+
+* **`compiler`**: Documentation about the compiler technology, codenamed
+  Testarossa. 
+
+Much documentation about OpenJ9 is best considered in tandem with the
+documentation from [Eclipse OMR](https://github.com/eclipse/omr/tree/master/doc),
```

Comment:
maybe "documentation about" should be in the square brackets too, since the link goes to the doc, not the project